src/renderer/router/index.js

```diff
@@ -15,9 +14,8 @@ import Channel from '../views/Channel/Channel.vue'
 import Watch from '../views/Watch/Watch.vue'
 import Hashtag from '../views/Hashtag/Hashtag.vue'
 
-Vue.use(Router)
-
-const router = new Router({
+const router = createRouter({
+  history: createWebHistory(),
```

Comment:
We are currently using hash history (default in the old vue-router https://v3.router.vuejs.org/api/#mode), so you should do the same here.